src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ConnectivityTests/AADConnectionTest.cs

```diff
@@ -161,10 +161,10 @@ public static void AADPasswordWithWrongPassword()
             string[] credKeys = { "Password", "PWD" };
             string connStr = DataTestUtility.RemoveKeysInConnStr(DataTestUtility.AADPasswordConnectionString, credKeys) + "Password=TestPassword;";
 
-            AggregateException e = Assert.Throws<AggregateException>(() => ConnectAndDisconnect(connStr));
+            SqlException e = Assert.Throws<SqlException>(() => ConnectAndDisconnect(connStr));
```

Comment:
:+1:

Reply:
It does categorize under Breaking Change though, right?

Reply:
Technically, it could... Though this could be considered a bug fix as well...